vue-frontend/src/components/ios-app-development/BlogSection.vue

```diff
@@ -0,0 +1,238 @@
+        :centeredSlides="true"
+        :loop="false"
+        :creativeEffect="{
+          limitProgress: 5,
+          loop: false,
+          next: {
+            translate: [800, 0, 0],
+          },
+          prev: {
+            translate: [0, 0, -100],
+          },
+        }"
+        :breakpoints="{
+          '450': {
+            creativeEffect: {
+              limitProgress: 5,
+              next: {
+                translate: [450, 0, 0],
+              },
+              prev: {
+                translate: [0, 0, -100],
+              },
+            },
+          },
+        }"
+        :slidesPerView="1.7"
+        :spaceBetween="10"
+        :modules="modules"
+      >
+        <swiper-slide
+          v-for="(blog, index) in blogs"
+          :key="index"
+          class="tw-mt-8"
+        >
+          <div
+            :class="blog.className"
+            class="tw-w-60 xs:tw-w-80 tw-h-[350px] tw-drop-shadow-md tw-text-left tw-cursor-pointer"
+          >
+            <img
+              @click="openBlog(blog.link, 'tap_ios_blog_section')"
+              :src="blog.image"
+              :srcset="`${blog.image} 400w`"
+              :alt="blog.title"
+              class="tw-h-44 tw-w-80 tw-object-fit xs:tw-object-cover"
+              loading="lazy"
+            />
+            <div
+              @click="openBlog(blog.link, 'tap_ios_blog_section')"
+              class="tw-w-60 xs:tw-w-80 tw-h-32 xs:tw-h-28 tw-bg-white tw-flex tw-flex-col tw-justify-center tw-px-8"
+            >
+              <h2
+                class="tw-font-inter-medium tw-text-basetw-leading-6 tw-tracking-[-4%] tw-text-black-core/[0.87]"
+              >
+                {{ blog.title }}
+              </h2>
+
+              <span
+                class="tw-mt-2 tw-font-inter-regular tw-text-sm tw-tracking-[-0.72px] lg:tw-leading-[1.3125rem] v2-canopas-gradient-text"
+              >
+                {{ blog.readTime }}
+              </span>
+            </div>
+          </div>
+        </swiper-slide></swiper
+      >
+    </div>
+
+    <!-- Mobile UI End -->
+    <!-- Desktop UI start -->
+    <div class="tw-mt-8 md:tw-block tw-hidden">
+      <swiper
+        :slidesPerView="2.5"
+        :spaceBetween="10"
+        :centeredSlide="true"
+        :modules="modules"
+        :speed="1000"
+        :autoplay="{
+          delay: 2000,
+          disableOnInteraction: false,
+        }"
+        :breakpoints="{
+          '768': {
+            slidesPerView: 2,
+          },
+          '900': {
+            slidesPerView: 2,
+          },
+          '1200': {
+            slidesPerView: 2.5,
+          },
+          '1400': {
+            slidesPerView: 3.5,
+          },
+
+          '2400': {
+            slidesPerView: 2.5,
+          },
+        }"
+      >
+        <swiper-slide
+          v-for="(blog, index) in iosblog"
+          :key="index"
+          class="tw-cursor-pointer tw-relative tw-flex tw-flex-col tw-pl-4 sm:tw-pl-8"
```

Comment:
Can you set this card design like the second row in the last section of the contribution page, I think it's better than this.

eg. The first and last card should be cut equally, not just the last card, cut both cards equally.